### ISSUE TITLE
Add raw method

### ISSUE THREAD

**Issue by fregante** [ENT8R/wifiradio#4]
I was about to use this module for my [personal shortcuts app](https://github.com/fregante/marshall-multi-room) but I noticed that it only has a handful of methods. I wouldn't ask you to support more, but generic get/set methods would be useful:

```js
radio.get('sys.info.friendlyname').then(name => {
  console.log({name})
});
radio.set('sys.audio.volume', 5).then(result => {
  console.log({result})
});
```

**Comment by ENT8R** [comment]
Very good idea, thank you for proposing it!
I already implemented it and will release this addition as `v0.4.0` on npm now 🎉

**Comment by fregante** [comment]
Great, thanks!
